test/e2e/specs/all-features-confirmation/recordset/presentation.spec.js

```diff
@@ -416,6 +502,100 @@ describe('View recordset,', function() {
                 });
             });
 
+            it("apply sorting on different columns.", function () {
```

Comment:
After talking to @RFSH about this, we think this `it()` should be split into 2 separate `it()` cases. One testing sorting in ascending order and one testing sorting in descending order.

This should be done by changing this structure to something like:
```js
describe("", function () { 
  forEach(accommodationParams.sortedData, function (data) {
    (function (k) {
      it("do ascending test", function () {...});
      it("do descending test", function () {...});
    }(j));
  }
})
```